### PR TITLE
Finished up what was leftover from yesterday.

### PR DESCRIPTION
**Pull request by brandonto**
One thing to note is that I have not actually ran the samples yet. The instructions to run the samples are ported from the dev portal and may or may not be outdated.

### CODE REVIEW

File: docs/_tutorials/persistence-with-queues.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The two sections below can be added to your pom.xml to configure it use the Paho
...
 
 NOTE: If you use the default 'MqttConnectOptions', or set 'MqttConnectOptions.cleanSession' to 'true', as done in the publish/subscribe tutorial, then a Non-Durable (a.k.a. Temporary Endpoint) queue will automatically be created on the Solace message router when the client adds a QoS 1 subscription. Queues are used to store QoS 1 messages providing persistence for the 'MqttClient'. A Non-Durable queue is removed when the 'MqttClient' disconnects, which mean the Solace message router will not retain any messages for the client after it disconnects. Setting the 'MqttConnectOptions.cleanSession' to 'false' will create a Durable queue which will retain messages even after the client disconnects. You can learn more about Solace queue durability from the Endpoint Durability section of [Solace Messaging Platform Feature Guide – Working with Guaranteed Messages](https://sftp.solacesystems.com/Portal_Docs/#page/Solace_Messaging_Platform_Feature_Guide/Working_with_Guaranteed_Messages.html#).
```
Comment (mdspielman) [inline docs/_tutorials/persistence-with-queues.md]: You're missing the target from the Working with Guaranteed Messages link. Target should be _top.

File: docs/_tutorials/persistence-with-queues.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ This tutorial uses Quality of Service (QoS) level of 1 (equivalent to Solace “
 
 A topic filter in MQTT differs from a Solace SMF topic subscription. Users can learn more about the differences between the two in the Topic Names and Filters section of [MQTT Specification Conformance – Operational Behavior](https://sftp.solacesystems.com/Portal_Docs/#page/MQTT_Specification_Conformance/4_Operational_behavior.html).
```
Comment (mdspielman) [inline docs/_tutorials/persistence-with-queues.md]: Ditto here. Need target _top

**Review by mdspielman — CHANGES_REQUESTED** [review]
A few comment found and tracked as review notes. Also, to build the samples, I believe you'll need to update the "build.gradle" so that the build will output the desired wrapper scripts in ./build/staged/bin

**Comment by brandonto** [comment]
Made some changes. Let me know if there are any additional changes required.

**Pull request merged**